components/Hero/src/index.scss

```diff
@@ -171,6 +192,43 @@
     --denhaag-hero-content-column-start: var(--denhaag-hero-home-content-column-start-xs, 0);
     --denhaag-hero-content-column-end: var(--denhaag-hero-home-content-column-end-xs, 12);
   }
+
+  .denhaag-hero--theme-page .denhaag-hero__container {
+    padding-block-start: var(--denhaag-hero-theme-page-container-padding-block-start-s);
+  }
+
+  .denhaag-hero--theme-page .denhaag-hero__content {
+    min-height: var(--denhaag-hero-theme-page-content-min-height-s);
+    height: var(--denhaag-hero-theme-page-content-height-s);
+  }
+
+  .denhaag-hero--theme-page .denhaag-hero__title {
+    font-size: var(--denhaag-hero-theme-page-title-font-size-s);
+  }
+
+  .denhaag-hero--theme-page .denhaag-hero__text {
+    font-size: var(--denhaag-hero-theme-page-text-font-size-s);
+  }
+}
+
+@media (min-width: 430px) {
+  .denhaag-hero--theme-page {
+    height: var(--denhaag-hero-theme-page-height-s);
+  }
+
+  .denhaag-hero--theme-page .denhaag-hero__text {
+    font-size: var(--denhaag-hero-theme-page-text-font-size-m);
```

Comment:
Why not do the same as the other hero's and use the power of CSS variables?

For example:

`@media (min-width: 430px) { 
 .denhaag-hero--theme-page { 
   --denhaag-hero-theme-page-height: var(--denhaag-hero-theme-page-height-s);
   --denhaag-hero-theme-page-text-font-size:  var(--denhaag-hero-theme-page-text-font-size-m); 
 }
}
`

This way u shorten the CSS output and use the way CSS variables are meant to be,